src/main/java/com/alkemy/ong/controller/NewsController.java

```diff
@@ -92,7 +92,7 @@
                     description = DocumentationMessages.NEWS_CONTROLLER_RESPONSE_404_DESCRIPTION)})
     @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE,
             produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<NewsDto> create(@RequestBody @Valid NewsDto dto) {
+    public ResponseEntity<NewsDto> create(@RequestBody @Valid NewsDto dto) throws Exception{
```

Comment:
When it's necessary to modify a class in the main project, it's recommended to create a tk of type bug. :)